templates/settings/admin.php

```diff
@@ -1,4 +1,7 @@
 <?php
+// TODO: uncomment once there are custom styles in the admin settings entrypoint
+// style('contacts', 'contacts-index');
+// style('contacts', 'contacts-admin-settings');
```

Comment:
smells like dead code

Reply:
I can remove it again. 

I just dropped a hint for a future maintainer that is wondering why custom styles won't work. In case we pimp the admin settings in the future.